moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -47,17 +48,22 @@ include_directories(
 ## Library to process realtime twist or joint commands ##
 #########################################################
 
+# This library provides a way of loading parameters for servo
+add_library(${SERVO_PARAM_LIB_NAME} SHARED src/servo_parameters.cpp)
+set_target_properties(${SERVO_PARAM_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+ament_target_dependencies(${SERVO_PARAM_LIB_NAME} ${THIS_PACKAGE_INCLUDE_DEPENDS})
+
 # This library provides an interface for sending realtime twist or joint commands to a robot
```

Comment:
Let's specify that it's a C++ interface, not a ROS2 topic interface:

`# This core library provides a C++ interface for sending realtime twist or joint commands to a robot`